public/terms-of-service.tex

```diff
@@ -0,0 +1,150 @@
+\documentclass[11pt]{article}
```

Comment:
It should be moved to another folder like `docs/` or `_docs` in order to not be part of the build (making it smaller).